internal/k8s/ako_init.go

```diff
@@ -570,3 +595,16 @@ func SyncFromNodesLayer(key string, wg *sync.WaitGroup) error {
 	restlayer.DeQueueNodes(key)
 	return nil
 }
+
+//Controller Specific method
+func (c *AviController) InitializeNameSpaceSync() {
+	nsLabelToSync := lib.GetLabelToSyncNameSpace()
+	utils.AviLog.Debugf("Initializing NameSpace Sync. Received namespace label: [%s]", nsLabelToSync)
+	utils.InitializeNSSync(nsLabelToSync)
+	nsFilterObj := utils.GetGlobalK8NSObj()
+	if !nsFilterObj.EnableMigration {
```

Comment:
Should we check if EnableMigration is true and then call InitializeNSSync () ?

Reply:
Taken care. We are now checking labels are there or not. If present, only then calling intializeNSSync to initialise global structure.